LayoutTests/imported/w3c/web-platform-tests/css/css-values/signs-abs-computed-expected.txt

```diff
@@ -205,4 +205,31 @@ PASS abs(-1turn) should be used-value-equivalent to 1turn
 PASS sign(10px - 1em) should be used-value-equivalent to 0; fontSize=10px
 PASS sign(10px - 2em) should be used-value-equivalent to -1; fontSize=10px
 PASS calc(sign(10%) * 100px) should be used-value-equivalent to 100px
+PASS calc(2.5 - sign(41px - 2em) / 2) should be used-value-equivalent to 2
+PASS calc(2.5 - sign(40px - 2em) / 2) should be used-value-equivalent to 2.5
+PASS calc(2.5 - sign(39px - 2em) / 2) should be used-value-equivalent to 3
+PASS calc(3 + sign(42px - 2em)) should be used-value-equivalent to 4
+PASS calc(3 + sign(40px - 2em)) should be used-value-equivalent to 3
+PASS calc(3 + sign(38px - 2em)) should be used-value-equivalent to 2
+PASS calc(50px + 100px * sign(42px - 2em)) should be used-value-equivalent to 150px
+PASS calc(50px + 100px * sign(40px - 2em)) should be used-value-equivalent to 50px
+PASS calc(50px + 100px * sign(38px - 2em)) should be used-value-equivalent to -50px
+PASS calc(90deg + 30deg * sign(42px - 2em)) should be used-value-equivalent to 120deg
+PASS calc(90deg + 30deg * sign(40px - 2em)) should be used-value-equivalent to 90deg
+PASS calc(90deg + 30deg * sign(38px - 2em)) should be used-value-equivalent to 60deg
+PASS calc(5s + 15s * sign(42px - 2em)) should be used-value-equivalent to 20s
+PASS calc(5s + 15s * sign(40px - 2em)) should be used-value-equivalent to 5s
+PASS calc(5s + 15s * sign(38px - 2em)) should be used-value-equivalent to -10s
+FAIL calc(100dpi + 20dpi * sign(42px - 2em)) should be used-value-equivalent to 120dpi assert_not_equals: calc(100dpi + 20dpi * sign(42px - 2em)) isn't valid in 'image-resolution'; got the default value instead. got disallowed value undefined
+FAIL calc(100dpi + 20dpi * sign(40px - 2em)) should be used-value-equivalent to 100dpi assert_not_equals: calc(100dpi + 20dpi * sign(40px - 2em)) isn't valid in 'image-resolution'; got the default value instead. got disallowed value undefined
+FAIL calc(100dpi + 20dpi * sign(38px - 2em)) should be used-value-equivalent to 80dpi assert_not_equals: calc(100dpi + 20dpi * sign(38px - 2em)) isn't valid in 'image-resolution'; got the default value instead. got disallowed value undefined
+FAIL calc(3fr + 1fr * sign(42px - 2em)) should be used-value-equivalent to 4fr assert_not_equals: calc(3fr + 1fr * sign(42px - 2em)) isn't valid in 'grid-template-rows'; got the default value instead. got disallowed value "none"
+FAIL calc(3fr + 1fr * sign(40px - 2em)) should be used-value-equivalent to 3fr assert_not_equals: calc(3fr + 1fr * sign(40px - 2em)) isn't valid in 'grid-template-rows'; got the default value instead. got disallowed value "none"
+FAIL calc(3fr + 1fr * sign(38px - 2em)) should be used-value-equivalent to 2fr assert_not_equals: calc(3fr + 1fr * sign(38px - 2em)) isn't valid in 'grid-template-rows'; got the default value instead. got disallowed value "none"
```

Comment:
We failed these 6 not because of calc() issues, but rather because it is using the property `image-resolution` for the 3 resolution tests and is assuming that `grid-template-rows` supports flex units, which the implementation in WebKit seemingly does not.